doc/findinfiles.rst

```diff
@@ -1,27 +1,58 @@
-:orphan:
+####
+Find
+####
 
-#############
-Find in Files
-#############
-
-The **Find in Files** pane allows to you perform full-text, recursive content search in a user-defined scope, with advanced features to filter your results.
+The **Find** pane allows you to search the content of files in a user-defined location, with advanced features to filter your results.
 
 .. image:: images/find_in_files/find_in_files_inprogress.png
    :alt: Spyder Find in Files panel, with search results shown per-file
 
 |
 
 
+
+===================
+Using the Find pane
+===================
+
+To search for text in the Find pane, enter it in the combobox in the top left and press the search button.
+This will allow you to view and navigate through all the occurrences of your search text in your working directory.
+You can expand or collapse the search results to view the matches in each file.
+Clicking on a match will automatically open the file and highlight the line where the text was found.
+
+[GIF showing search and cliking file]
+
+If you want to change the scope of your search, select another directory, project or file in the :guilabel:`Search in` menu.
+The locations that you select for your search will be stored in the list so you can access them easily in the future.
+To erase all of these saved directories, select the :guilabel:`Clear the list" option from the dropdown menu in the :guilabel:`Search in` field.
+
+[GIF showing chosing new directory and directory stored in the list]
+
+
+
 =======================
 Choosing search options
 =======================
 
-The :guilabel:`Find in Files` tool allows you to select from a number of options to enable searches as broad or refined as you need:
+You can select from a number of options to enable searches as broad or refined as you need.
+
+To enable case sensitivity, controlling whether matches are returned with a different case than your search text, toggle the :guilabel:`Aa` button on.
```

Comment:
```suggestion
To enable case sensitivity, controlling whether results match exactly the search text you provided, toggle the :guilabel:`Aa` button on.
```

I think this is clearer. I mean, from the current description (and without seeing the gif) I couldn't figure out what this functionality is supposed to do (as a non-native speaker).

Reply:
@CAM-Gerlach, I think you missed this one.

Reply:
This does look just a bit unclear. However, the problem with this suggestion is that it now doesn't refer to case at all, it states whether "results match exactly" which refers to fuzzy search, not case sensitive, so I'm not sure how this is clearer and I don't know if there is another word in English could be used here to mean the same thing; we could explicitly say upper and lower case but that still involves the word "case". Ideas?

Reply:
Can you try to explain what makes it less clear @ccordoba12 ? I think there might be a simpler way of explaining it but your suggestion doesn't really says what the case sensitive does.

Reply:
When I use case sensitive searches, I want to find exact matches for the text I introduced. For instance, I use them to find results just for `Foo`, instead of `foo` and `Foo`.

Reply:
I see. The issue is that without using the word case, the phrasing above would be generally understood to refer to a fuzzy text search, not anything to do with case-sensitivity.

What about this?

```suggestion
To enable case sensitivity, which only returns matches in the same alphabet case (upper or lower) as your search text, toggle the :guilabel:`Aa` button on.
```

Reply:
That's more complicated for me than the original version. What about using "different capitalization" instead of "case" (somehow)?

The thing is if we use the word "case" to describe what "case sensitive" means, then things are kind of circular.

Reply:
Okay, I see. How about this?

```suggestion
To enable case sensitivity, which only returns matches with identical capitalization to your search text, toggle the :guilabel:`Aa` button on.
```

Reply:
I think the last suggestion sounds good. @ccordoba12 what do you think?